.travis.yml

```diff
@@ -1,31 +1,54 @@
+dist: xenial
 language: python
 python:
+  - "3.7"
+  - "3.6"
   - "3.5"
```

Comment:
@aubreypc `python type hinting` is only supported in versions `3.6` and above. Would you like me to get rid of versions `3.6` and below?